### PR TITLE
Make MRTCore work in framework package

### PR DESCRIPTION
**Pull request by huichen123**
I will stamp ProjectReunionFrameworkPackage in aggregation pipeline.

### CODE REVIEW

**Review by axelandrejs — APPROVED** [review]

**Review by rohanp-msft — APPROVED** [review]

File: dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <ItemGroup Condition="'$(OutputType)' == 'Exe'">
     <Reference Include="Microsoft.ApplicationModel.Resources.winmd">
```
Comment (rohanp-msft) [inline dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props:17]: ><Reference Include="Microsoft.ApplicationModel.Resources.winmd"> [](start = 4, length = 64)

What about this? Is adding this reference OK?

Comment (huichen123) [reply dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props:17]: we need to have this reference. or we will get build break.

Comment (rohanp-msft) [reply dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props:17]: won't the winmd ref be added by the nuget package install? just like the DLL refs, etc.?

Comment (huichen123) [reply dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props:17]: you added that. So I guess it's a question for you? :)

Comment (rohanp-msft) [reply dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props:17]: currently, when the nuget package is installed, it does add the winmd ref

**Comment by Scottj1s** [comment]
thanks Hui!

**Pull request merged**